src/index.html

```diff
@@ -9,10 +9,42 @@
     <title>Calendar</title>
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="src/styles/main.scss"
```

Comment:
The stylesheet link references a SCSS file (`main.scss`). SCSS files need to be compiled into CSS before they can be used in an HTML document. Make sure to compile the SCSS file into a CSS file and update the `href` attribute to point to the resulting CSS file.